e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js

```diff
@@ -82,3 +82,9 @@ Scenario('Judiciary creates multiple orders for the case', async (I, caseViewPag
     await orderFunctions.assertOrder(I, caseViewPage, orders[i], i + 1);
   }
 });
+
+Scenario('Judiciary adds allocated judge', async (I, caseViewPage, allocatedJudgeEventPage) => {
+  await caseViewPage.goToNewActions(config.applicationActions.allocatedJudge);
+  await allocatedJudgeEventPage.enterAllocatedJudge('Moley');
+  await I.completeEvent('Save and continue');
```

Comment:
this test seems to be missing any assertion!

Reply:
whoops will add :)